packages/backend/src/auto-peering/service.ts

```diff
@@ -56,8 +68,107 @@ export async function createAutoPeeringService(
   }
 
   return {
-    acceptPeeringRequest: (args: PeeringRequestArgs) =>
-      acceptPeeringRequest(deps, args)
+    acceptPeeringRequest: (args) => acceptPeeringRequest(deps, args),
+    initiatePeeringRequest: (args) => initiatePeeringRequest(deps, args)
+  }
+}
+
+async function initiatePeeringRequest(
+  deps: ServiceDependencies,
+  args: InitiatePeeringRequestArgs
+): Promise<Peer | AutoPeeringError> {
+  const { peerUrl } = args
+
+  const asset = await deps.assetService.get(args.assetId)
+
+  if (!asset) {
+    return AutoPeeringError.UnknownAsset
+  }
+
+  const outgoingHttpToken = uuid()
+
+  const peeringDetailsOrError = await sendPeeringRequest(deps, peerUrl, {
+    ilpConnectorAddress: deps.config.ilpConnectorAddress,
+    staticIlpAddress: deps.config.ilpAddress,
+    asset: { code: asset.code, scale: asset.scale },
+    httpToken: outgoingHttpToken,
+    maxPacketAmount: Number(args.maxPacketAmount),
+    name: deps.config.instanceName
+  })
+
+  if (isAutoPeeringError(peeringDetailsOrError)) {
+    return peeringDetailsOrError
+  }
+
+  const peerOrError = await deps.peerService.create({
+    maxPacketAmount: args.maxPacketAmount,
+    name: args.name ?? peeringDetailsOrError.name,
+    assetId: asset.id,
+    staticIlpAddress: peeringDetailsOrError.staticIlpAddress,
+    http: {
+      incoming: {
+        authTokens: [peeringDetailsOrError.httpToken]
+      },
+      outgoing: {
+        authToken: outgoingHttpToken,
+        endpoint: peeringDetailsOrError.ilpConnectorAddress
+      }
+    }
```

Comment:
This is where I thought it would be 
```js
   http: {
      incoming: {
        authTokens: [incomingHttpToken] //generated here and communicated to the peer
      },
      outgoing: {
        authToken: peeringDetailsOrError.httpToken, //generated by the peer for me to use when I send a request to the peer
        endpoint: peeringDetailsOrError.ilpConnectorAddress
      }
    }
```

Reply:
I see what you mean, but is there a big difference in this case? I was thinking about conceptually, the peer just presents _its_ connection details, including the token it will be sending for payments

Reply:
I thought for the use case, where I'm requesting to auto-peer but the other Rafiki for some reason already has me as a peer, that other Rafiki can just respond with the token it already has on file for me. Otherwise, it would have to update its peering relationship with the token I'm now presenting. But I guess that is also an option.

Reply:
I'm actually already doing the updating: https://github.com/interledger/rafiki/pull/1849/files#diff-7b03b8ad222a5bb774fbc5e8afa3e8588dd1a28da1acabd65f33222d263e325dR98-R115

This allows for the receiving peer to update any of the arguments it receives from the first peer

Reply:
Since this should only be used with fake money and in sandbox environments, I guess that is fine.